configuration/concepts/reviewofsystems.csv

```diff
@@ -0,0 +1,75 @@
+063cb6f4-f54d-4ee6-878b-6c683cacac8a,,,Platypnea,Platypnée,Platypnea,Platypnée,,,Misc,N/A,,,,,,,,,,,
+7629a17f-9fc1-4fb6-a766-059cc88620a3,,,Productive Cough,Expectoration,Productive Cough,Expectoration,,,Misc,N/A,,,,,,,,,,,
+78d7751f-5ff3-4cef-b9a7-4c81d55fa5ed,,,Respiratory,Respiratoire,Respiratory,Respiratoire,,,Misc,Coded,,Toux (R05);Hemoptysie (R04.2);Dyspnee (R06.0);Orthopnée;Platypnée;Expectoration;Autre,,,,,,,,,
+97c382db-646b-4ee8-8bc4-fb630a9fc1e6,,,Productive Cough Details,Détails de la toux productive,Productive Cough Details,Détails de la toux productive,,,Misc,Text,,,,,,,,,,,
+1885d33d-9384-4b41-a2a4-4550fea7bf2d,,,Respiratory Set,Respiratoire set,Respiratory Set,Respiratoire set,,,ConvSet,N/A,,,Respiratoire;Détails de la toux productive;Autres observations,,,,,,,,
+00e0aad9-7fa0-48af-9000-003bfd616875,,,Paroxysmal Nocturnal Dyspnea,Dyspnée paroxystique Nocturne,Paroxysmal Nocturnal Dyspnea,Dyspnée paroxystique Nocturne,,,Misc,N/A,,,,,,,,,,,
+25ec3417-f125-4e8c-8862-7f50123b4f81,,,Dyspnea with Exertion,Dyspnée d'Effort,Dyspnea with Exertion,Dyspnée d'Effort,,,Misc,N/A,,,,,,,,,,,
+05aa444c-d7e4-4b7f-a737-9cfbbe321d53,,,Lower Extremity Edema,Oédeme des membres inférieurs,Lower Extremity Edema,Oédeme des membres inférieurs,,,Misc,N/A,,,,,,,,,,,
+3fc8f995-6db1-43af-a277-f8c03a879e43,,,Claudication,Claudication,Claudication,Claudication,,,Misc,N/A,,,,,,,,,,,
+c7f7721d-8f2f-4bcf-a9b7-aff9470c9e6f,,,Hematochezia,Hématochézie,Hematochezia,Hématochézie,,,Misc,N/A,,,,,,,,,,,
+d34da7b5-2ede-4231-8abf-a5fb27d69608,,,Dyschezia,Dyschésique,Dyschezia,Dyschésique,,,Misc,N/A,,,,,,,,,,,
+188fbe84-7fe4-459b-9dc5-689fc16c2e46,,,Cardiovascular,Cardiovasculaire,Cardiovascular,Cardiovasculaire,,,Misc,Coded,,Douleur thoracique (R07.4);Dyspnée paroxystique Nocturne;Dyspnée d'Effort;Oédeme des membres inférieurs;Claudication;Des palpitations;Syncope;Autre,,,,,,,,,
+2059f006-1c3f-4ab4-8d05-7c6d51c662d5,,,Cardiovascular Set,Ensemble cardiovasculaire,Cardiovascular Set,Ensemble cardiovasculaire,,,ConvSet,N/A,,,Cardiovasculaire;Autres observations,,,,,,,,
+1b5792c6-9fb0-445f-a588-823b8b5d091d,,,Melena,Méléna,Melena,Méléna,,,Misc,N/A,,,,,,,,,,,
+45b8d250-4f1a-47c2-ab1e-41205d41d675,,,Gastrointestinal,Gastrointestinal,Gastrointestinal,Gastrointestinal,,,Misc,Coded,,Dysphagie (R13);Douleurs abdominales;La nausée;Vomissement (R11);Hematemese (K92.0);Méléna;Diarrhee (K52.9);Constipation;Hématochézie;Dyschésique;Autre,,,,,,,,,
+660f5c64-d23c-4b98-a9fd-b507834fcab0,,,Gastrointestinal Set,Gastrointestinal set,Gastrointestinal Set,Gastrointestinal set,,,ConvSet,N/A,,,Gastrointestinal;Autres observations,,,,,,,,
+6788f174-9be5-40af-b938-379cf3745c88,,,Head,Tête,Head,Tête,,,Misc,Coded,,Cephalee (R51);Vertiges;Masse;Convulsion (R56.8);Autre,,,,,,,,,
+212f798e-7625-459f-8512-a88606408126,,,Head Set,Tête set,Head Set,Tête set,,,ConvSet,N/A,,,Tête;Autres observations,,,,,,,,
+4305753b-937e-4a22-b299-e1680af09919,,,Urgency,Impériosité,Urgency,Impériosité,,,Misc,N/A,,,,,,,,,,,
```

Comment:
Urgenturie (if this means sudden need to urinate)

Reply:
Done